apps/backend/local-stack/bin/run-native-stt.sh

```
@@ -1,42 +1,87 @@
-    tar xjf "$MODEL_DIR/moonshine.tar.bz2" -C "$MODEL_DIR"
-    rm -f "$MODEL_DIR/moonshine.tar.bz2"
+# Batch engine (optional on the host): whisper-server must be installed
+# separately; without it the service still streams and reports batch
+# unavailable via /v1/capabilities.
+if command -v whisper-server >/dev/null 2>&1; then
+    if [ ! -f "$BATCH_FILE" ]; then
+        echo "⚠ whisper batch model missing at $BATCH_FILE — batch will be unavailable"
+        echo "  (fetch it with: bun stack/fetch_models.ts --entry stt-whisper-tiny)"
+    else
+        echo "Starting whisper.cpp batch server on 127.0.0.1:$WHISPER_PORT ..."
+        WHISPER_PORT="$WHISPER_PORT" \
```

Shellcheck (0.11.0)

[warning] This assignment is only seen by the forked process.

(SC2097)